server/helpers/mail.js

```diff
@@ -0,0 +1,28 @@
+import nodemailer from 'nodemailer';
+import Joi from '@hapi/joi';
+import pug from 'pug';
+import path from 'path';
+import MailSchema from '../validations/mail';
+import joiOptions from '../validations/joiOptions';
+import config from '../config';
+
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: config.EMAIL,
+    pass: config.EMAIL_PASSWORD,
+  },
+});
+
+const sendMail = async function sendMail(type, mailOptions, variables) {
```

Comment:
put a check over here, if the environment is local, then just log instead of actually sending mail, this will help in preventing testing mails to users

Reply:
done